Client/Frontend/Fakespot/Views/HighlightsCardView.swift

```diff
@@ -0,0 +1,75 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import ComponentLibrary
+import Common
+import UIKit
+
+struct HighlightsCardViewModel {
+    let cardA11yId: String = AccessibilityIdentifiers.Shopping.HighlightsCard.card
+    let footerTitle: String
+    let footerActionTitle: String
+    let footerA11yTitleIdentifier: String = AccessibilityIdentifiers.Shopping.HighlightsCard.footerTitle
+    let footerA11yActionIdentifier: String = AccessibilityIdentifiers.Shopping.HighlightsCard.footerAction
+
+    var footerModel: ActionFooterViewModel {
+        return ActionFooterViewModel(title: footerTitle,
+                                     actionTitle: footerActionTitle,
+                                     a11yTitleIdentifier: footerA11yTitleIdentifier,
+                                     a11yActionIdentifier: footerA11yActionIdentifier)
+    }
+}
+
+class HighlightsCardView: UIView, ThemeApplicable {
```

Comment:
I guess this view is FakeSpot specific?

Reply:
Yes, it is. It's in a Fakespot folder. Do you prefer it being renamed to show it's for Fakespot?